package.json

```diff
@@ -28,12 +28,12 @@
     "test:watch": "jest --watch",
     "lint": "eslint \"{src,test,scripts}/**/*.{js,jsx,ts,tsx}\" --quiet",
     "lint:fix": "npm run lint -- --fix",
-    "extract": "formatjs extract 'src/**/*.{js,jsx,ts,tsx}' --out-file temp.json --flatten --id-interpolation-pattern '[sha512:contenthash:base64:6]' --preserve-whitespace",
-    "compile": "formatjs compile 'temp.json' --out-file src/i18n/locales/en-US.json",
+    "extract": "formatjs extract \"src/**/*.{js,jsx,ts,tsx}\" --out-file temp.json --flatten --id-interpolation-pattern '[sha512:contenthash:base64:6]' --preserve-whitespace",
+    "compile": "formatjs compile \"temp.json\" --out-file src/i18n/locales/en-US.json",
     "manage-translations": "npm run extract && npm run compile && rimraf temp.json",
     "build": "preval-build-info-cli && gulp build && electron-builder",
     "contributors": "all-contributors",
-    "reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write --require-pragma \"**/*.{js,jsx,scss}\"",
```

Comment:
when I added this in, without the path, `prettier` was not found - maybe that's fixed now, just want to confirm this

Reply:
Please ensure its not picking up a global prettier while verifying this

Reply:
> Please ensure its not picking up a global prettier while verifying this

I'm almost sure I don't have a global `prettier` package installed... Nevertheless let me check

Reply:
Just checked
![image](https://user-images.githubusercontent.com/37463445/178647912-22c18256-e6d4-4ad5-9f46-7b767c4bac9b.png)

I don't have any global dependency installed. (As this is a new node installation through nvs, and I frequently run our clean script)